app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
Add rails validations to model, if not you will get a DB exception instead of more expressive rails validation errors

Reply:
The validations are in the book-test branch.